pkg/api/apikey.go

```diff
@@ -65,7 +65,7 @@ func (hs *HTTPServer) AddAPIKey(c *m.ReqContext, cmd m.AddApiKeyCommand) Respons
 	cmd.Key = newKeyInfo.HashedKey
 
 	if err := bus.Dispatch(&cmd); err != nil {
-		if err.Error() == "Negative value for SecondsToLive" {
+		if err == m.ErrInvalidApiKeyExpiration {
```

Comment:
Should be `models.ErrInvalidApiKeyExpiration` so the import at the top shouldn't be aliased. Think we decided that, but can't find any documentation about it.